lib/dependencies/index.ts

```diff
@@ -24,8 +25,13 @@ export async function getDependencies(
   }
   let command = options.command || 'python';
   const includeDevDeps = !!(options.dev || false);
-  let baseargs: string[] = [];
 
+  // handle poetry projects by parsing manifest & lockfile and return a dep-graph
+  if (path.basename(targetFile) === 'pyproject.toml') {
```

Comment:
I think would be nice to add a type for `'pyproject.toml'`
to prevent coupled string